homeworks/ekaterina.sharova_rogerosha/PokemonProject/style.css

```diff
@@ -0,0 +1,3 @@
+.colortext {
```

Comment:
Please add text properties (`font-size`, `line-heght`, `font-weight`, `color`,...) from the design to link, header, pragraph
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/1e726351-4ae8-4f5e-9cdd-e97e2f7d8a80)